src/main/IndexToFST.java

```diff
@@ -23,41 +21,6 @@
 import org.apache.lucene.util.fst.PositiveIntOutputs;
 import org.apache.lucene.util.fst.Util;
 
-/*
-2
-4
-8
-16
-32
-64
-128
-256
-512
-1024
-2048
-4096
-8192
-16384
-32768
-65536
-131072
-262144
-524288
-1048576
-2097152
-4194304
-8388608
-16777216
-33554432
-67108864
-134217728
-268435456
-536870912
-1073741824
-2147483648
-4294967296
```

Comment:
wait but what will we do without this handy reference!

Reply:
Hahaha!!  At one point the RAM -> fst size behavior was quantized on these power-of-two boundaries, but now it is a smoothly monotonic behaving knob of a beastie.